docs/develop/sso-in-office-add-ins.md

```diff
@@ -1,69 +1,86 @@
 ---
-title: Enable single sign-on for Office Add-ins
-description: 'Learn how to enable single sign-on for Office Add-ins using common Microsoft personal, work, or education accounts.'
-ms.date: 09/03/2021
+title: Enable single sign-on (SSO) in an Office Add-in
+description: 'Learn the key steps to enable single sign-on (SSO) for your Office Add-in using common Microsoft personal, work, or education accounts.'
+ms.date: 01/07/2022
 ms.localizationpriority: high
 ---
 
-# Enable single sign-on for Office Add-ins
+# Enable single sign-on (SSO) in an Office Add-in
 
-Users sign in to Office (online, mobile, and desktop platforms) using either their personal Microsoft account or their Microsoft 365 Education or work account. You can take advantage of this and use single sign-on (SSO) to authorize the user to your add-in without requiring the user to sign in a second time.
+Users sign in to Office (online, mobile, and desktop platforms) using either their personal Microsoft account or their Microsoft 365 Education or work account. You can take advantage of this and use single sign-on (SSO) to authenticate and authorize the user to your add-in without requiring the user to sign in a second time.
 
 ![An image showing the sign-in process for an add-in.](../images/sso-for-office-addins.png)
 
+## How SSO works at runtime
+
+The following diagram shows how the SSO process works. The blue elements represent Office or the Microsoft identity platform. The gray elements represent the code you write and include the client-side code (task pane) and the server-side code for your add-in.
+
+![A diagram that shows the SSO process.](../images/sso-overview-diagram.svg)
+
+1. In the add-in, your JavaScript code calls the Office.js API [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken_options_). If the user is already signed in to Office, the Office host will return the access token with the claims of the signed in user.
+2. If the user is not signed in, the Office host application opens a dialog box for the user to sign in. Office redirects to the Microsoft identity platform to complete the sign-in process.
+3. If this is the first time the current user has used your add-in, they are prompted to consent.
+4. The Office host application requests the **access token** from the Microsoft identity platform for the current user.
+5. The Microsoft identity platform returns the access token to Office. Office will cache the token on your behalf so that future calls to **getAccessToken** simply return the cached token.
```

Comment:
```suggestion
5. The Microsoft identity platform returns the access token to Office. Office caches the token on your behalf so that future calls to **getAccessToken** simply return the cached token.
```